sqlup-mode.el

```diff
@@ -67,19 +67,12 @@
 this mode's logic will be evaluated.")
 
 (defconst sqlup-eval-keywords
-  '((postgres "EXECUTE"))
-"List of keywords introducing eval strings, organised by dialect.")
+  '((postgres "EXECUTE" "format("))
```

Comment:
I see. And I'm guessing we here have `format(` instead of `EXECUTE format(` because sqlup just goes back one symbol?